### DISCUSSION TITLE
Installed capacity germany

### DISCUSSION THREAD

**Discussion by dblndr** [electricitymaps/electricitymaps-contrib#5974] [General]
How often do you update the installed capacity? E.g. solar should be in the 76gw ballpark, but is 69gw.

which sources are consulted?

**Comment by Ddock5** [comment]
Ireland is shown as **no solar** but the installed PV capacity is over 700 MW which could be 15% of production on a clear summer day.

**Reply by VIKTORVAV99** [reply]
Unrelated to the current discussion but feel free to open a issue about it being missing or if you have a data source for it update it directly.

**Comment by VIKTORVAV99** [comment]
Updating the capacities is so far a manual job (we are working on solutions though) so that is usually something that we rely on contributors to do as we simply can't check all our data sources manually all the time.

The data source that is used for each zone and data type should be listed in the [DATA_SOURCES.md](https://github.com/electricitymaps/electricitymaps-contrib/blob/master/DATA_SOURCES.md) file.
